aws-ecsfargate-terraform/main.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.14.0"
+  required_version = ">= 1.1"
```

Comment:
Guessing this is the minimum version required for terraform to support the moved syntax?

Reply:
Yep!